snuba/api.py

```diff
@@ -344,37 +356,37 @@ def sdk_distribution(validated_body, timer):
 if application.debug or application.testing:
```

Comment:
Seems like you could make a reasonable argument that the table name references below should come from `get_local_table_name` rather than `get_table_name`, but not a strong opinion here.

Reply:
fixed